src/Stryker.CLI/Stryker.CLI/CLIOptions.cs

```diff
@@ -1,20 +1,13 @@
 ﻿using McMaster.Extensions.CommandLineUtils;
-using Microsoft.CodeAnalysis.CSharp;
-using Stryker.Core.Baseline;
 using Stryker.Core.Options;
-using Stryker.Core.Reporters;
-using Stryker.Core.TestRunners;
-using System;
+using Stryker.Core.Options.Options;
 using System.Collections.Generic;
 using System.Linq;
-using System.Text;
 
 namespace Stryker.CLI
 {
     public static class CLIOptions
     {
-        private static readonly StrykerOptions _defaultOptions = new StrykerOptions();
-
         public static readonly CLIOption<string> ConfigFilePath = new CLIOption<string>
```

Comment:
ConfigFilePath has no strykerInput to put the helptext

Reply:
That's because ConfigFilePath is not a stryker input ;) It is used in the commandline only. I have hardcoded the helptext there.